scss/_global.scss

```diff
@@ -20,35 +20,50 @@ $global-width: rem-calc(1200) !default;
 /// @type Number
 $global-lineheight: 1.5 !default;
 
-/// Colors used for buttons, callouts, links, etc. There must always be a color called `primary`.
+/// Colors available by using `get-color()`.
```

Comment:
Please precise that unline the previous `get-color`, it these colors should not be used inside components.